editor/filesystem_dock.cpp

```diff
@@ -1575,6 +1575,17 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> &p_selected
 			OS::get_singleton()->shell_open(String("file://") + dir);
 		} break;
 
+		case FILE_OPEN_IN_SHELL: {
```

Comment:
I find "OPEN_IN_SHELL" a little bit misleading, as it looks like it's going to open a terminal. OPEN_WITH_DEFAULT_PROGRAM would be better IMHO.